src/language/__tests__/schema-kitchen-sink.graphql

```diff
@@ -17,6 +17,7 @@ type Foo implements Bar {
   four(argument: String = "string"): String
   five(argument: [String] = ["string", "string"]): String
   six(argument: InputType = {key: "value"}): Type
+  seven(argument: Int = null): Type
```

Comment:
Yeah this is probably confusing as an *example* because I agree that it's a little strange to make a default value of `null` to rule out the "not provided" (aka undefined) case, but it's certainly possible now, so it's worth testing as an *edge case*.